### ISSUE TITLE
[Suggestion] Auto updater should only check for latest releases

### ISSUE THREAD

**Issue by Eddga** [TCNOco/TcNo-Acc-Switcher#359]
I noticed when being on the latest Release version ([2023-06-27_01](https://github.com/TCNOco/TcNo-Acc-Switcher/releases/tag/2023-06-27_01) I'm asked to update to the currently latest Pre-Release version [2023-07-05_00](https://github.com/TCNOco/TcNo-Acc-Switcher/releases/tag/2023-07-05_00) even though it contains known bugs.
It would be a better UX if I'd either not be offered to update to a non-Release version at all or (better) could opt-in for Beta/Pre-Release versions or at least opt-out of them.
Otherwise I'd always be bothered with being asked to update to a newer version that contains bugs that probably break the program for me.

**Comment by Wontell** [comment]
> I noticed when being on the latest Release version ([2023-06-27_01](https://github.com/TCNOco/TcNo-Acc-Switcher/releases/tag/2023-06-27_01) I'm asked to update to the currently latest Pre-Release version [2023-07-05_00](https://github.com/TCNOco/TcNo-Acc-Switcher/releases/tag/2023-07-05_00) even though it contains known bugs. It would be a better UX if I'd either not be offered to update to a non-Release version at all or (better) could opt-in for Beta/Pre-Release versions or at least opt-out of them. Otherwise I'd always be bothered with being asked to update to a newer version that contains bugs that probably break the program for me.

What kind of bugs it contains? Since i went for the latest version, maybe i should downgrade

**Comment by Eddga** [comment]
@Wontell read the release notes: https://github.com/TCNOco/TcNo-Acc-Switcher/releases/tag/2023-07-05_00

> This version does have a known issue:
> You are unable to use the forget account option. Do not update to this if you use this feature. This will be addressed in the next version.

**Comment by TCNOco** [comment]
Even for testing I need to create this kind of feature... Running the `updater.exe 2024-08-25_00` for example would help a lot with beta testing. Currently, the [updates file](https://tcno.co/Projects/AccSwitcher/api/update?debug&v=) doesn't have anything about betas, but I can implement this.

However, at the same time I don't really plan on pushing betas... Or at least haven't. It's usually feature update > live. I should include a `Beta` option, so users can test newer versions for a few days before full release.

This should be a planned feature and I'll get to it when I can :)

**Comment by Eddga** [comment]
@TCNOco Thanks for your work. Yeah well in that case.. I dunno don't stress to hard on it. I just started to use the app at a point where it had a known bug so maybe it was just bad luck. 😅
Side note: In your latest release notes I suppose you forgot to remove the known bug about forgetting accounts as later in the notes you say it's fixed.

**Comment by TCNOco** [comment]
Yes, thanks I missed that but it should be removed :) Not sure when it'll update for everyone...
